### ISSUE TITLE
[bug] Moon detects .git repositories and node_modules inside of a project folder as separate projects.

### ISSUE THREAD

**Issue by NikolaRHristov** [moonrepo/moon#257]
When using moonrepo with workspaces the folders inside the main project folder also get detected as projects. Such folders include .git, etc.

```yaml
projects:
    - "app/**"
```

This is the logfile

```
✔ Successfully synced 22 projects
pass SetupToolchain (skipped)
pass SyncSystemProject(playform) (0s)
pass SyncSystemProject(info) (0s)
pass SyncSystemProject(objects) (0s)
pass SyncSystemProject(pages) (0s)
pass SyncSystemProject(remotes) (0s)
pass SyncSystemProject(pack) (0s)
pass SyncSystemProject(public) (0s)
pass SyncSystemProject(origin) (0s)
pass SyncSystemProject(workflows) (0s)
pass SyncNodeProject(blank-web-starter) (skipped)
pass SyncSystemProject(scripts) (0s)
pass SyncSystemProject(tags) (0s)
pass SyncSystemProject(refs) (0s)
pass SyncSystemProject(git) (0s)
pass SyncSystemProject(styles) (0s)
pass SyncSystemProject(layouts) (0s)
pass SyncSystemProject(logs) (0s)
pass SyncSystemProject(heads) (0s)
pass SyncSystemProject(app) (0s)
pass SyncSystemProject(src) (0s)
pass SyncSystemProject(hooks) (0s)
pass SyncSystemProject(github) (0s)
```

**Comment by milesj** [comment]
@NikolaRHristov I can add some safe guards, but you should probably avoid using `**` and just use `apps/*`.

**Comment by milesj** [comment]
Thanks for the report, this will be fixed here: https://github.com/moonrepo/moon/pull/259

**Comment by milesj** [comment]
This is fixed in v0.11.
